python/graphscope/nx/tests/test_copy_on_write.py

```diff
@@ -389,7 +398,7 @@ def test_node_boundary(self):
 
     def test_edge_boundary(self):
         ret = nx.builtin.edge_boundary(self.SG, [1, 2])
-        assert ret == [[2, 3], [1, 3]]
+        assert ret == [[1, 3]]
```

Comment:
ditto

Reply:
There was a bug in edge_boundary and fixed,  `[[1, 3]]` is the correct answer.